src/main/kotlin/no/nav/tilleggsstonader/sak/vedtak/barnetilsyn/TilsynBarnBeregnYtelseSteg.kt

```diff
@@ -90,11 +91,16 @@ class TilsynBarnBeregnYtelseSteg(
     ) {
         val andelerTilkjentYtelse = beregningsresultat.perioder.flatMap {
             it.beløpsperioder.map { beløpsperiode ->
+                val satstype = Satstype.DAG
+                val day = beløpsperiode.dato.dayOfWeek
+                feilHvis(day == DayOfWeek.SATURDAY || day == DayOfWeek.SUNDAY) {
+                    "Skal ikke opprette perioder som begynner på en helgdag for satstype=$satstype"
+                }
```

Comment:
Ikke så krise men greit å være konsistent
```suggestion
                val dag = beløpsperiode.dato.dayOfWeek
                feilHvis(dag == DayOfWeek.SATURDAY || dag == DayOfWeek.SUNDAY) {
                    "Skal ikke opprette perioder som begynner på en helgdag for satstype=$satstype"
                }
```

Reply:
✅